### PR TITLE
Supported XLayer Netwok

### PR DESCRIPTION
**Pull request by zjg555543**
Supported for XLayer Tesetnet and Mainnet

### CODE REVIEW

**Comment by mandrigin** [comment]
@zjg555543 please resolve the conflicts and I will re-review, overall looks okay

**Comment by mandrigin** [comment]
also please check

https://github.com/0xPolygonHermez/cdk-erigon?tab=readme-ov-file#dynamic-chain-configuration

File: params/config.go [hunk 1]
```diff
@@ -66,7 +66,8 @@ var (
 	HermezESTestGenesisHash            = libcommon.HexToHash("0x8c630b598fab24a99b59cdd8257f41b35d0aca992f13cd381c7591f5e89eec58")
 	HermezCardonaGenesisHash           = libcommon.HexToHash("0x676c1a76a6c5855a32bdf7c61977a0d1510088a4eeac1330466453b3d08b60b9")
 	HermezCardonaInternalGenesisHash   = libcommon.HexToHash("0x7311011ce6ab98ef0a15e44fe29f7680909588322534d1736361daa678543038")
-	X1TestnetGenesisHash               = libcommon.HexToHash("0x22a8085892b367833bd7431fa5a90ff6b5d3769167cdaa29ce8571d07bc8f866")
+	XLayerTestnetGenesisHash           = libcommon.HexToHash("0x0ffb92e130f1acaabd8b12aa1bb409b46561ef7568cb8aa7eb8d102a6ab76566")
```
Comment (zjg555543) [inline params/config.go]: @mandrigin Is the hash come form the getBlockByNumber form the 0 block? Or other

**Comment by zjg555543** [comment]
> @zjg555543 please resolve the conflicts and I will re-review, overall looks okay

Done

**Review by hexoscott — APPROVED** [review]

**Pull request merged**